client_cli/src/main.rs

```diff
@@ -888,6 +915,74 @@ mod asset {
             Ok(())
         }
     }
+
+    #[derive(clap::Args, Debug)]
+    pub struct SetKeyValue {
+        /// AssetId for the Store asset (in form of `asset##account@domain_name')
+        pub asset_id: AssetId,
+        /// The key for the store value
+        #[clap(long)]
+        pub key: Name,
+        /// Enter the value to be associated with the specified key.
+        /// This value can be of various types, including boolean(true),
+        /// numeric(1_u32), IP address(127.0.0.1), public key(ed01207233BFC89DCBD68C19FDE6CE6158225298EC1131B6A130D1AEB454C1AB5183C0)
+        /// or JSON where the key is the type({"Vec":[{"String":"a"},{"String":"b"}]}).
+        #[clap(long)]
+        pub value: ValueArg,
+    }
+
+    impl RunArgs for SetKeyValue {
+        fn run(self, context: &mut dyn RunContext) -> Result<()> {
+            let Self {
+                asset_id,
+                key,
+                value: ValueArg(value),
+            } = self;
+
+            let set = iroha_client::data_model::isi::SetKeyValue::asset(asset_id, key, value);
+            submit([set], UnlimitedMetadata::default(), context)?;
+            Ok(())
+        }
+    }
+    #[derive(clap::Args, Debug)]
+    pub struct RemoveKeyValue {
+        /// AssetId for the Store asset (in form of `asset##account@domain_name')
+        pub asset_id: AssetId,
```

Comment:
Without `#[clap(long)]` `asset_id` becomes a positional argument:

```sh
iroha_client_cli asset remove-key-value <asset-id> --key <key>
```

Reply:
In principle the following are acceptable: 
### Both positional;
```
iroha_client_cli asset remove-key-value <key> <asset-id>
```

### Both `--long` and `-s`hort for both arguments. 
```
iroha_client_cli asset remove-key-value -k <key> -a <asset-id>
```
It should be obvious and probably not much of a head scratcher, plus it only takes a few lines of code. 

### Asset as positional to asset:

```
iroha_client_cli asset <asset-id> remove-key-value <key>
```

Imo this makes more sense but might require reworking how the client_cli works.